### DISCUSSION TITLE
ctx mem_size

### DISCUSSION THREAD

**Discussion by desaiIshan00** [ggml-org/whisper.cpp#2130] [Q&A]
I'm trying to build a personal AI voice assistant using GPT-2. When I run the command ./talk -p santa, it gives this error:
"gpt2_model_load: ggml ctx size = 384.74 MB ggml_new_object: not enough space in the context's memory pool (needed 403447760, available 403425792) Segmentation fault"
I've tried increasing the mem_size in the ggml.c file, but that's not working. How can I solve this? Thank you.

**Comment by ggerganov** [comment]
`talk` is quite outdated. It's better to use `talk-llama` as a starting point - it is regularly maintained
